### PR TITLE
EPMRPP-36738. Fix filter query value for testItem pages & Add interrupted test type for history line

### PR DESCRIPTION
**Pull request by AmsterGet**

### CODE REVIEW

File: app/src/controllers/testItem/sagas.js [hunk 1]
```diff
@@ -96,7 +96,9 @@ function* fetchTestItems() {
         'filter.eq.launch': launchId,
         'filter.eq.parent': !noChildFilter ? parentId : undefined,
         'filter.level.path': sizePath,
-        'filter.under.path': noChildFilter ? parentId : undefined,
+        'filter.under.path': noChildFilter
+          ? itemIds.filter((item) => item !== launchId.toString()).join('.')
```
Comment (DiscoElevator) [inline app/src/controllers/testItem/sagas.js]: Is it necessary to use `toString` here?

Comment (AmsterGet) [reply app/src/controllers/testItem/sagas.js]: `itemIds` is represented as an array of strings, but `launchId` may be a `Number` according to new API

Comment (DiscoElevator) [reply app/src/controllers/testItem/sagas.js]: Maybe we should convert all ids to number in selectors to reflect API changes?

Comment (AmsterGet) [reply app/src/controllers/testItem/sagas.js]: We get `itemIds` from `location` in `state`
Should I convert this items to numbers in appropriate selector?
image

Comment (AmsterGet) [reply app/src/controllers/testItem/sagas.js]: Done

**Pull request merged**